tests/codemods/test_enable_jinja2_autoescape.py

```diff
@@ -117,6 +117,42 @@ def test_autoescape_enabled(self, tmpdir):
     def test_autoescape_callable(self, tmpdir, code):
         self.run_and_assert(tmpdir, code, code)
 
+    def test_kwargs_unpacked(self, tmpdir):
+        input_code = (
+            expexted_output
+        ) = """
+        import jinja2
+        env = jinja2.Environment(**kwargs)
```

Comment:
I would've thought a more telling test case would be
`env = jinja2.Environment(**{"autoescape": True})` but I think it will trigger in all cases.